application/python/read_accelerometers_afif.py

```diff
@@ -0,0 +1,19 @@
+from dobot import DobotDriver

```

Comment:
Please name the file to reflect that it is an example instead of part of your name.